.github/workflows/test.yml

```diff
@@ -86,6 +79,7 @@ jobs:
       - run: go mod download
       - env:
           TF_ACC: "1"
+          IS_TEST_ACCOUNT: "1"
```

Comment:
This is used to let acceptance tests know that their running against the test apono account.